frontend/src/pages/Admin/Questions/AdminQuestionAdd.js

```diff
@@ -22,10 +23,11 @@ const AdminQuestionAdd = () => {
     newObj.description = formValues.question;
     choices.map((choice, index) => {
       if (`values[${index}]` === answer) {
-        return (choice.is_answer = true);
+        choice.is_answer = true;
       } else {
-        return (choice.is_answer = false);
+        choice.is_answer = false;
       }
+      return choices;
```

Comment:
Above map structure is a bit weird, why are we returning choices inside the callback.
Can you try like, if it works with the output u want:
```
choices.map((choice, i) => {
  return {...choice, is_answer: `values[${index}]` === answer}
})
```